usaspending_api/common/mixins.py

```diff
@@ -254,6 +256,70 @@ class SuperLoggingMixin(LoggingMixin):
 
     events_logger = logging.getLogger("events")
 
+    """Mixin to log requests - customized to disable DB logging, remove this method to re-enable"""
```

Comment:
Ah, ok...let me make sure I understand....we're overriding a third party library that was logging to the db so that we can grab the info it conveniently collects for us and log it to a file instead?